app/src/main/java/com/example/sharingang/items/Item.kt

```diff
@@ -46,6 +46,8 @@ data class Item(
     /** Id of the user that created the item */
     val userId: String? = null,
 
+    var rated: Boolean = false,
```

Comment:
If possible, try to have keep the fields of Item and User immutable.
You can easily make a copy of an item and change a simple field:
`item.copy(rated = true)`